### PR TITLE
[ADD] Expenses: reinvoicing expenses

### PR DESCRIPTION
**Pull request by larm-odoo**
Original Expenses doc too long, breaking off the reinvoicing expenses to accounting section into its own doc, and making necessary updates to version 17 (original was written for 16). Also updating to new doc guidelines.

[Project Task Card](https://www.odoo.com/odoo/project/3835/tasks/4078268?cids=3) for this doc.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @hojo-odoo - this is ready for a first-round review. Thanks!

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:5]:
```suggestion
If expenses are tracked on customer projects, they can be automatically charged back to the
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:9]:
```suggestion
<expenses/reinvoice-approve>`, before the accounting department :ref:`posts the journal entries
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:15]:
```suggestion
   Approving expenses, posting expenses to accounting, and reinvoicing expenses on :abbr:`SOs (Sales Orders)` is
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:41]:
```suggestion
selection for :guilabel:`Re-Invoice Expenses`:

- :guilabel:`None`: The expense category is not be re-invoiced
- :guilabel:`At cost`: The expense category invoices expenses at their real cost.
- :guilabel:`At sales price`: The expense category invoices at the price set on the sales order.
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:52]:
```suggestion
Once a :abbr:`SO (Sales Order)` is attached to an expense report, and the report is saved, a
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:55]:
```suggestion
Next, select the :guilabel:`Analytic Distribution` the expense is posted to. Multiple accounts
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:72]:
```suggestion
   :guilabel:`Analytic Distribution` line, for 50% each.
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:101]:
```suggestion
Distribution` section is populated for every expense line.
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:114]:
```suggestion
The :abbr:`SO (Sales Order)` is **only** updated *after the journal entries are posted*. Once the
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:125]:
```suggestion
Select the expense report and click the :icon:`fa-money` :guilabel:`Sales Orders` smart button to open
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:132]:
```suggestion
   a list displaying all the :abbr:`SOs (Sales Orders)` associated with that expense report. Click on
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (hojo-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:141]:
```suggestion
Next, click :guilabel:`Create Invoice`, and a :guilabel:`Create invoices` pop-up window appears.
```

**Review by hojo-odoo — APPROVED** [review]
Hey @larm-odoo,

Great job on this doc. I'm approving with a handful of suggestions.

Thanks,
John

**Comment by larm-odoo** [comment]
Hi @ksc-odoo  - this is ready for a final review! I have fixed all merge errors, but just be aware, if other docs get uploaded before your review, there might be another one or two (which I'll fix when I get this back!)

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:54]: When I had clicked 'Create Report' to see the 'Sales Order' smart button, that field disappeared. I had to go back (via the browser) to see this field reappear, which also removed the 'Sales Order' smart button. Could this Analytic Distribution step be done on the 'Report' page, via the 'Analytic Distribution' column? I was able to see the same popover there... 

Either way, I think this particular portion needs some slight reworking to clarify things a bit more for the user.

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:58]:
```suggestion
:guilabel:`Analytic` pop-over window. Click :guilabel:`Add a line`, then select the desired
```
opting for the use of the word "desired" instead of "next" because I wouldn't want readers to assume they have to pick the very next account available, should that not be their intention.

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:62]:
```suggestion
total of all selected accounts equals `100%`.
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:67]:
```suggestion
   the estimate, a meeting is held at the office location to discuss the project.
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [reply content/applications/finance/expenses/reinvoice_expenses.rst:54]: Upon further thought, is it even necessary to mention a 'Sales Order' smart button if it doesn't need to be interacted with yet? Without that mention, things would flow seamlessly, as intended. Especially considering that you instruct them to create and submit the report in the following section...Something to think about...

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:103]: Do you think it's important to mention that this 'Approve' button only appears after clicking 'Submit to manager'? I think that would be very helpful...

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:113]:
```suggestion
The :abbr:`SO (Sales Order)` is **only** updated *after* the journal entries are posted. Once the
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:124]:
```suggestion
Select the expense report, and click the :icon:`fa-money` :guilabel:`Sales Orders` smart button to
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:142]:
```suggestion
a :guilabel:`Down payment (fixed amount)`. Then, click
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:143]:
```suggestion
:guilabel:`Create Draft Invoice`. Doing so creates a draft invoice for the customer. Click
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (ksc-odoo) [inline content/applications/finance/expenses/reinvoice_expenses.rst:144]:
```suggestion
:guilabel:`Confirm` to confirm the invoice, and the customer is invoiced for the expenses.
```

**Review by ksc-odoo — APPROVED** [review]
hey @larm-odoo -- just finished my Final Review of this PR. There are a handful of comments/feedback for you to consider. If you have any questions, don't hesitate to reach out. Always happy to help. Once you implement the necessary adjustments, and feel like th is one is ready, you can feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses/reinvoice_expenses.rst:51]: Correct- I checked this, and since you wouldn't want to do that for an individual record, I moved this sentence and rephrased it to the 'submit' section (made more sense that way.).

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (larm-odoo) [reply content/applications/finance/expenses/reinvoice_expenses.rst:54]: lol- yes, I just commended on the first comment you made =D

I moved this whole mention to the 'submit' section since it makes much more sense over there.

**Comment by larm-odoo** [comment]
This is ready for a tech review, @samueljlieber!

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses/reinvoice_expenses.rst:84]: Please update media folder to use an underscore rather than a hyphen :)
```suggestion
.. image:: reinvoice_expenses/reinvoice-expense.png
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses/reinvoice_expenses.rst:108]:
```suggestion
.. image:: reinvoice_expenses/analytic-dist.png
```

File: content/applications/finance/expenses/reinvoice_expenses.rst
Comment (samueljlieber) [inline content/applications/finance/expenses/reinvoice_expenses.rst:143]:
```suggestion
.. image:: reinvoice_expenses/so-details.png
```

**Review by samueljlieber — COMMENTED** [review]
Hi @larm-odoo, just a couple quick fixes for some images. Thank you for your work on this Expenses PR!

Please address these mandatory fixes before merge, thanks!
.....
@robodoo delegate=larm-odoo

**Comment by larm-odoo** [comment]
@robodoo r+

**Comment by larm-odoo** [comment]
@robodoo retry

**Comment by larm-odoo** [comment]
@robodoo r+

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
